### PR TITLE
feat: enabled agreement score service

### PR DESCRIPTION
**Pull request by MagicTearsAsunder**

### CODE REVIEW

File: annotation/app/errors.py [hunk 1]
```diff
@@ -156,5 +156,5 @@ def agreement_score_service_error_handler(
 ):
     return JSONResponse(
         status_code=400,
-        content={"detail": f"Agreement score service error. {exc.exc_info}"},
+        content={"detail": f"Agreement score service error. {exc}"},
     )
```
Comment (spiridonovfed) [inline annotation/app/errors.py:160]: I usually do:
try:
    ...
except SomeException as exc:
    raise HTTPException(status_code=400, detail="Agreement score service error. {exc}"

But it's up to you

File: annotation/app/schemas/tasks.py [hunk 2]
```diff
@@ -152,6 +152,7 @@ class AgreementScoreServiceInput(BaseModel):
     task_id: int = Field(..., example=1)
     s3_file_path: str = Field(...)
     s3_file_bucket: str = Field(...)
+    s3_tokens_path: str = Field(...)
     manifest_url: str = Field(...)
```
Comment (spiridonovfed) [inline annotation/app/schemas/tasks.py:156]: s3_file_path: str = Field(...)
is the same as:
s3_file_path: str

There's no point of using "= Field(...)" unless you use some additional attributes, like 'example' in line 152

File: annotation/app/tasks/resources.py [hunk 3]
```diff
@@ -920,7 +920,11 @@ def finish_task(
 
             # TODO extensive coverage, annotators, manifest_url
             extensive_coverage = task.jobs.extensive_coverage
-            if extensive_coverage and extensive_coverage > 1:
+            if (
+                extensive_coverage
+                and extensive_coverage > 1
+                and AGREEMENT_SCORE_ENABLED == "true"
```
Comment (spiridonovfed) [inline annotation/app/tasks/resources.py:926]: it's up to you, but this way seems more Pythonic to me:

#.env
AGREEMENT_SCORE_ENABLED=0

#code.py
if AGREEMENT_SCORE_ENABLED:
    ....

File: annotation/app/tasks/services.py [hunk 4]
```diff
@@ -539,21 +540,38 @@ def evaluate_agreement_score(
...
+        .all()
+    )
+    tasks_intersection_pages.append(task)
+
     s3_file_path, s3_file_bucket = get_file_path_and_bucket(
         task.file_id, tenant, token.token
     )
+    # s3_file_path, s3_file_bucket = (
+    #     f"files/{task.file_id}/{task.file_id}.pdf", tenant
+    # )
```
Comment (spiridonovfed) [inline annotation/app/tasks/services.py:562]: looks like redundant code

**Review by spiridonovfed — CHANGES_REQUESTED** [review]

File: annotation/app/errors.py (hunk 1 above)
Comment (MagicTearsAsunder) [reply annotation/app/errors.py:160]: This is a common error handler

File: annotation/app/tasks/resources.py (hunk 3 above)
Comment (MagicTearsAsunder) [reply annotation/app/tasks/resources.py:926]: I think this is less explicit

File: annotation/app/tasks/services.py (hunk 4 above)
Comment (MagicTearsAsunder) [reply annotation/app/tasks/services.py]: fixed

File: annotation/app/schemas/tasks.py (hunk 2 above)
Comment (MagicTearsAsunder) [reply annotation/app/schemas/tasks.py]: fixed

**Review by spiridonovfed — APPROVED** [review]

**Pull request merged**